src/main/kotlin/me/partlysanestudios/partlysaneskies/features/dungeons/HealerAlert.kt

```diff
@@ -49,12 +47,18 @@ object HealerAlert {
         }
         return false
     }
-    fun run() {
-        if (!PartlySaneSkies.config.healerAlert){
+
+    @SubscribeEvent
+    fun onTick(event: TickEvent.ClientTickEvent) {
```

Comment:
Hi so this was intentional that way we could have all of our tick classes in one place. It should be called from the PartlySaneStudios.onTick

Reply:
oh,
most are missing tho

Reply:
Uh



oh

Reply:
also this feels weird doesnt it?